ios/FluentUI/Resources/Localization/he.lproj/Localizable.strings

```diff
@@ -181,7 +181,4 @@
 "Presence.Unknown" = "לא ידוע";
 
 /* Presence - Blocked status */
-"Presence.Blocked" = "חסום/ה";
-
-/* AvatarView - Format string for overflow avatar. Format: "+<OverflowCount>". Example: "+5" */
-"AvatarView.OverflowFormat" = "ועוד %@";
+"Presence.Blocked" = "חסום/ה";
```

Comment:
what happened here?

Reply:
There was a merge conflict and the AvatarView localized string populated again. I just removed it so it matched the other files